src/java/com/google/tagpost/spanner/SpannerService.java

```diff
@@ -150,4 +168,18 @@ private void initDb() {
     }
     return commentListBuilder.build();
   }
+
+  private TagStats convertResultToTagStats(ResultSet resultSet) {
```

Comment:
Can this method be static?

Reply:
Yes, I also changed other helper methods to be static